README.md

```diff
@@ -59,9 +60,11 @@ supported.
 Do not set the Cloud Run region you'd like to deploy to through the `--region` flag in the `gcloud run` commands.
 Instead, as mentioned above, do so by setting the `run/region` gcloud property.
 
-In addition to setting a default Cloud Run region, make sure to deploy to the fully managed platform on Cloud Run. You
-can achieve this by setting the `run/platform` gcloud property to `managed` or passing in the `--platform=managed` flag
-to your `gcloud run` commands.
+### README location
+If you wish to parse a README file located somewhere other than the root directory, you can include the README's location
+in a `config.yaml` file in the root directory, using the key `readme`.
 
-If comment code tags aren't added to your README, the program will fall back to reasonable defaults to build and deploy
-your sample to Cloud Run based on whether your sample is java-based and doesn't have a Dockerfile or isn't.
+For example:
+```text
+readme: ../README.md
```

Comment:
Is this path relative to the config.yaml? Relative to cwd? 

How does sst determine what the root directory is?

Reply:
This path is relative to the root directory of the sample directory specified on the command line; I'll update the README so this is more clear. (It will also work if the user specifies an absolute directory.) If you think a different path would make more sense, I can change it.

Reply:
As is, the README still isn't clear. It says "root directory", but doesn't define what that is. I think what you really men to say is _target_ directory - it's relative to the directory you passed to sst (which is also where the `config.yaml` can be found)